bcda/bcdacli/cli.go

```diff
@@ -323,7 +323,7 @@ func setUpApp() *cli.App {
 				// Generate new credentials
 				// TODO: GenerateClientCredentials interface to be updated as part of BCDA-1304
```

Comment:
I suppose you could remove this `TODO` now

Reply:
Good catch!